### PR TITLE
docs: Update reward-distribution.md

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
## Description

The existing comment in the ICS docs was inconsistent with the [comment for ConsumerRedistributionFraction in code](https://github.com/cosmos/interchain-security/blob/eddc0ca5b6781d58fa916056eb0b844447fc5d6e/proto/interchain_security/ccv/consumer/v1/consumer.proto#L41). Thanks @asalzmann for pointing this out

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary 
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [x] Confirmed the correct `docs:` prefix in the PR title
- [x] Confirmed all author checklist items have been addressed 
- [x] Confirmed that this PR only changes documentation
- [x] Reviewed content for consistency
- [x] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

### CODE REVIEW

**Review by mpoke — APPROVED** [review]

**Review by sainoe — APPROVED** [review]

File: docs/docs/features/reward-distribution.md [hunk 1]
```diff
@@ -12,6 +12,21 @@ The distributed reward tokens are IBC tokens and therefore cannot be staked on t
 
 Sending and distributing rewards from consumer chains to provider chain is handled by the `Reward Distribution` sub-protocol.
 
+## Note
+The ICS distribution system works by allowing consumer chains to send rewards to a module address on the provider called the `ConsumerRewardsPool`.
+There is a new transaction type called `RegisterConsumerRewardDenom`. This transaction allows consumer chains to register denoms to be used as consumer chain rewards on the provider.
+It costs `10 ATOM` to register a denom and the full amount is transferred to the community pool of the provider chain. Only denoms registered through this transaction are then transferred to the `FeePoolAddress` and distributed out to delegators and validators.
```
Comment (mpoke) [inline docs/docs/features/reward-distribution.md:18]: > costs `10 ATOM` to register a denom

This is a param on the Hub.

File: docs/docs/features/reward-distribution.md [hunk 2]
```diff
@@ -12,6 +12,21 @@ The distributed reward tokens are IBC tokens and therefore cannot be staked on t
...
+
+### Instructions for adding a denom
+The transaction must be carried out on the provider chain. Please use the `ibc/*` denom trace format.
```
Comment (mpoke) [inline docs/docs/features/reward-distribution.md:21]:
```suggestion
The transaction must be carried out on the provider chain. Please use the `ibc/*` denom trace format to register IBC denoms.
```

File: docs/docs/features/reward-distribution.md [hunk 3]
```diff
@@ -21,12 +36,12 @@ They are set at consumer genesis and `blocks_per_distribution_transmission`, `co
...
 
 ### `consumer_redistribution_fraction`
-The fraction of tokens sent from consumer to provider during distribution events. The fraction is a string representing a decimal number. For example "0.75" would represent 75%.
+The fraction of tokens allocated to the consumer redistribution address during distribution events. The fraction is a string representing a decimal number. For example "0.75" would represent 75%.
 
 :::tip
 Example:
 
-With `consumer_redistribution_fraction` set to `0.75` the consumer chain would send 75% of its block rewards and accumulated fees to the consumer chain and the remaining 25% to the provider chain every `n` blocks where `n == blocks_per_distribution_transmission`.
+With `consumer_redistribution_fraction` set to `0.75` the consumer chain would send 75% of its block rewards and accumulated fees to the consumer redistribution address, and the remaining 25% to the provider chain fee pool every `n` blocks where `n == blocks_per_distribution_transmission`.
```
Comment (mpoke) [inline docs/docs/features/reward-distribution.md:44]:
```suggestion
With `consumer_redistribution_fraction` set to `0.75` the consumer chain would send 75% of its block rewards and accumulated fees to the consumer redistribution address, and the remaining 25% to the provider chain every `n` blocks where `n == blocks_per_distribution_transmission`.
```

File: docs/docs/features/reward-distribution.md [hunk 4]
```diff
@@ -12,6 +12,21 @@ The distributed reward tokens are IBC tokens and therefore cannot be staked on t
 
 Sending and distributing rewards from consumer chains to provider chain is handled by the `Reward Distribution` sub-protocol.
 
+## Note
+The ICS distribution system works by allowing consumer chains to send rewards to a module address on the provider called the `ConsumerRewardsPool`.
+There is a new transaction type called `RegisterConsumerRewardDenom`. This transaction allows consumer chains to register denoms to be used as consumer chain rewards on the provider.
+The cost to register register a denom is configurable (`ConsumerRewardDenomRegistrationFee` chain param) and the full amount is transferred to the community pool of the provider chain. Only denoms registered through this transaction are then transferred to the `FeePoolAddress` and distributed out to delegators and validators.
```
Comment (smarshall-spitzbart) [inline docs/docs/features/reward-distribution.md:18]:
```suggestion
The cost to register register a denom is configurable (`ConsumerRewardDenomRegistrationFee` chain param) and the full amount of this fee is transferred to the community pool of the provider chain. Only denoms registered through this transaction are then transferred from the `ConsumerRewardsPool` to the `FeePoolAddress`, to be distributed out to delegators and validators.
```

**Pull request merged**